common.sh

```diff
@@ -142,6 +142,7 @@ export HIVE_DEPLOY_IMAGE="${HIVE_DEPLOY_IMAGE:-registry.ci.openshift.org/openshi
 export OPENSHIFT_CI=${OPENSHIFT_CI:-""}
 export OPENSHIFT_VERSION=${OPENSHIFT_VERSION:-""}
 if [[ -z "$OPENSHIFT_CI" ]]; then
+  # FIXME: the release stream no longer appears in the release image
```

Comment:
This isn't strictly true, it just isn't defined in the builds generated for CI:

```
2022-02-03 12:46:06 +++(common.sh:145): source(): echo registry.ci.openshift.org/ocp/release:4.10.0-0.nightly-2022-02-02-220834
2022-02-03 12:46:06 +++(common.sh:145): source(): sed 's/.*:\([[:digit:]]\.[[:digit:]]*\).*/\1/'
2022-02-03 12:46:06 ++(common.sh:145): source(): export OPENSHIFT_VERSION=4.10
2022-02-03 12:46:06 ++(common.sh:145): source(): OPENSHIFT_VERSION=4.10
```

So personally I'd say something like `FIXME: OPENSHIFT_VERSION should not be derived from the release URL, since this doesn't always work e.g in CI`

Reply:
This code _only_ runs in CI, per line 144.